Dockerfile

```diff
@@ -0,0 +1,434 @@
+ARG          DATAFED_DIR="/datafed"
+ARG DATAFED_INSTALL_PATH="$DATAFED_DIR/install"
+ARG            GCS_IMAGE="code.ornl.gov:4567/dlsw/datafed/gcs-ubuntu-focal"
+ARG            BUILD_DIR="$DATAFED_DIR/source"
+ARG              NVM_DIR="$DATAFED_DIR/.nvm"
+ARG              NVM_INC="$DATAFED_DIR/.nvm/versions/node/v13.14.0/include/node"
+ARG              NVM_BIN="$DATAFED_DIR/.nvm/versions/node/v13.14.0/bin"
+ARG              LIB_DIR="/usr/local/lib"
+
+FROM ubuntu:focal AS base
+
+FROM base AS dependencies
+
+ARG NVM_DIR
+ARG DATAFED_DIR
+ARG BUILD_DIR
+ARG DATAFED_INSTALL_PATH
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV DATAFED_DIR="${DATAFED_DIR}"
+
+RUN mkdir -p ${BUILD_DIR}
+RUN mkdir -p ${BUILD_DIR}/logs
+RUN mkdir -p ${BUILD_DIR}/repository/server
+RUN mkdir -p ${BUILD_DIR}/common/proto
+RUN mkdir -p ${NVM_DIR}
+
+WORKDIR ${BUILD_DIR}
+
+# Copy install scripts
+COPY ./scripts/dependency_install_functions.sh	${BUILD_DIR}/scripts/
+COPY ./scripts/dependency_versions.sh						${BUILD_DIR}/scripts/
+# COPY ./scripts/install_core_dependencies.sh			${BUILD_DIR}/scripts/
+# COPY ./scripts/install_repo_dependencies.sh			${BUILD_DIR}/scripts/
+# COPY ./scripts/install_ws_dependencies.sh				${BUILD_DIR}/scripts/
+# COPY ./scripts/install_gcs.sh ${BUILD_DIR}/scripts/
+# COPY ./scripts/install_authz_dependencies.sh ${BUILD_DIR}/scripts/
+COPY ./scripts/install_dependencies.sh ${BUILD_DIR}/scripts/
```

Comment:
Here we need to make sure that the same code in all the individual install scripts are actually being called so that all of the install scripts are being tested during the build process.